CharacterGenerator/Character.h

```diff
@@ -0,0 +1,31 @@
+//
+//  Character.h
+//  CharacterGenerator
+//
+//  Created by Henna on 8/2/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+@interface Character : NSObject
+
+//user generated
+@property (nonatomic) NSString * name;
```

Comment:
When declaring pointers to variables, put the `*` beside the variable name, i.e. `*name` so it's clear to the reader what it's a pointer to.